src/Sylius/Bundle/AdminBundle/Resources/config/services.xml

```diff
@@ -60,6 +60,16 @@
             <tag name="twig.extension" />
         </service>
 
+        <service id="Sylius\Bundle\AdminBundle\Twig\FilterExtension">
+            <argument type="service" id="Sylius\Bundle\AdminBundle\Storage\FilterStorage" />
+            <argument type="service" id="router" />
+            <tag name="twig.extension" />
+        </service>
+
+        <service id="Sylius\Bundle\AdminBundle\Storage\FilterStorage">
```

Comment:
Can't this service be private?

Reply:
I didn’t check that all services here are public. Good catch! Thanks 😃